### ISSUE TITLE
使用了DREAM模型后，为什么两个不同的人脸相似度反而提升了

### ISSUE THREAD

**Issue by songyy137222** [penincillin/DREAM#16]

**Comment by songyy137222** [comment]
两个相同人脸相似度是提升的，但提升特别小，我是用你提供的直接数据训练的DREAM模型（switching）

**Comment by penincillin** [comment]
Thanks for asking,  I get confused by question, you use the provided data or provide pretrained model?
What's more, English is prefered :)

**Comment by songyy137222** [comment]
I'm sorry.I get the DREAM model with the provided data. (python branch_train.py)
The test data is 256 dimensional vector with mobilefacenet.

**Comment by songyy137222** [comment]
@penincillin

**Comment by penincillin** [comment]
The data we provided is too little to train a useful model. The only purpose of providing this data is to make sure you can go through the whole pipeline of our code and have a idea of the correct format of the data.
To train a useful model, you should use much more data than **sample data** provided by us.

**Comment by songyy137222** [comment]
@penincillin Iget it,thanks!!!

**Comment by ysc703** [comment]
Hi, @songyy137222 
Have you got a good result by using branch_train?
I trained my own dataset with 58k images, by the cosine similarity of some pairs (same ID) becomes

**Comment by ysc703** [comment]
Hi, @songyy137222
Have you got a good result by using branch_train?
I trained the DREAM branch with my own dataset(58k images of 6k IDs), but the loss didn't decrease. 

Thanks!

**Comment by jolinlinlin** [comment]
@songyy137222 hello, Can you please tell me how to get 21-point of face to alignment?
